package.json

```
@@ -2,6 +2,10 @@
   "name": "fhir-spec-tools",
   "version": "0.0.1",
   "description": "Scripts and other shared tooling for working with the HL7® FHIR® standard",
+  "main": "build/index.js",
+  "files": [
+    "build/*"
```

```suggestion
    "build/data/*",
    "build/index.js"
```
This should export exactly what we need.

---

Also will want `build/scripts/types/*` or to move the types to the data directory. If we enable the "declarations" in the tsconfig.